test/com/gfredericks/test/chuck/clojure_test_test.clj

```diff
@@ -18,6 +18,24 @@
       (swap! c inc)
       (is (> @c 0)))))
 
+(deftest exception-detection-test
```

Comment:
This test was moved to [/test/com/gfredericks/test/chuck/exception_handling_test.clj](https://github.com/gfredericks/test.chuck/blob/master/test/com/gfredericks/test/chuck/exception_handling_test.clj) in master. This change should be done in that file, not here.